core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.spark.executor._
 import org.apache.spark.scheduler._
 import org.apache.spark.storage._
 import org.apache.spark._
+import org.apache.hadoop.hdfs.web.JsonUtil
```

Comment:
It doesn't look like this import is even used anywhere, so I think we can probably remove it.